src/components/Board/services/index.js

```diff
@@ -75,4 +75,40 @@ function removeCard(board, fromLane, card) {
   return { ...board, lanes: filteredLanes }
 }
 
-export { moveLane, moveCard, addLane, removeLane, renameLane, addCard, removeCard }
+function getCoordinates(event) {
+  if (event.destination === null) return {}
+
+  const laneSource = { fromPosition: event.source.index }
+  const laneDestination = { toPosition: event.destination.index }
+
+  if (isALaneMove(event.type)) {
+    return { source: laneSource, destination: laneDestination }
+  }
+
+  return {
+    source: { ...laneSource, fromLaneId: parseInt(event.source.droppableId) },
+    destination: { ...laneDestination, toLaneId: parseInt(event.destination.droppableId) }
+  }
+}
+
+function isALaneMove(type) {
```

Comment:
This function does not need to be exported, does it? I think it's only used inside the `getCoordinates` function.

Reply:
It is used in `BoardContainer` to check if it is a `card` or `lane` move.